README.md

```diff
@@ -124,6 +123,7 @@ We have flags for each chunk of the rewrite we release. These will set those pag
 
 - HOME_PAGE_REACT='true'
 - DIRECTORY_PAGE_REACT='true'
+- LISTING_DETAIL_PAGE_REACT='true'
```

Comment:
This doesn't match AC in DAH-990 which notes `REACT_RENTAL_LISTING_DETAILS`, but this one seems to line up with the other existing vars. Should we update the AC?